media-sound/lmms/lmms-1.2.0.recipe

```diff
@@ -48,7 +48,7 @@ REQUIRES="
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libfftw3f$secondaryArchSuffix
-	devel:libfluidsynth$secondaryArchSuffix >= 2
+	devel:libfluidsynth$secondaryArchSuffix
```

Comment:
Why remove the version requirement?

Reply:
Breaks hp pkg downloads. Is there a setting to override?

Reply:
> Breaks hp pkg downloads. Is there a setting to override?

This should have been fixed in https://github.com/haikuports/haikuporter/commit/2cc27775286313a45363dde360e0e6385adfa614.

Reply:
Please restore the version requirement